Extensions/ArtifactEngine/Engine/artifactEngine.ts

```diff
@@ -85,7 +85,22 @@ export class ArtifactEngine {
         }
         retryCount = retryCount ? retryCount : 0;
         if (item.itemType === models.ItemType.File) {
-            if (tl.match([item.path], this.patternList).length > 0) {
+            var pathToMatch = item.path.replace(/\\/g, '/');
+            var matchOptions = {
+                debug: false,
+                nobrace: true,
+                noglobstar: false,
+                dot: true,
+                noext: false,
+                nocase: true,
```

Comment:
does this mean case sensitive?

Reply:
yes, this should be false (case insensitive), will change